portal-web/test/functional/com/liferay/portalweb/macros/CompanyLogging.macro

```diff
@@ -15,7 +15,7 @@ definition {
 			value1 = "for group ${groupId}");
 	}
 
-	@summary = "Default summary"
+	@summary = "This will assert the company log isolated by removing ${additionalPortalInstanceName}"
```

Comment:
"This will assert the company log is isolated for a given company by asserting unique text contained in other company logs is not present"